src/Components/Game/index.tsx

```diff
@@ -1,20 +1,31 @@
 import React, { useState } from "react";
 import ColorPalette from "../ColorPalette";
 import "./style.css"
-import {COLOR_LIST, INITIAL_GAME_STATE, TOTAL_ATTEMPTS} from "../Utils/constants";
+import { COLOR_LIST, INITIAL_GAME_STATE, PEGS_PER_ATTEMPT, TOTAL_ATTEMPTS } from "../Utils/constants";
 import AttemptsContainer from "../AttemptsContainer";
-import { initializeBoard, initializeBoardHints } from "../Utils/util";
+import { generateSolution, initializeBoard, initializeBoardHints } from "../Utils/util";
+import swal from 'sweetalert';
```

Comment:
one thing i missed to tell is that, for better readability and convention we keep all the library imports at the top and then we put our local file imports